### ISSUE TITLE
Issue432 JSON Deser problem on unknown properties

### ISSUE THREAD

**Issue by uweschaefer** [factcast/factcast#436]
_From @uweschaefer on 2019-05-22 14:14:22 +0000 UTC_

_Copied from original issue: Mercateo/factcast#436_

**Comment by uweschaefer** [comment]
_From @codecov[bot] on 2019-05-22 14:16:27 +0000 UTC_

# [Codecov](https://codecov.io/gh/Mercateo/factcast/pull/436?src=pr&el=h1) Report
> Merging [#436](https://codecov.io/gh/Mercateo/factcast/pull/436?src=pr&el=desc) into [master](https://codecov.io/gh/Mercateo/factcast/commit/337080b9aae8712da4d9c2a37867a1c85e51b04c?src=pr&el=desc) will **not change** coverage.
> The diff coverage is `100%`.


```diff
@@           Coverage Diff           @@
##           master     #436   +/-   ##
=======================================
  Coverage   88.93%   88.93%           
=======================================
  Files          78       78           
  Lines        1870     1870           
  Branches      169      169           
=======================================
  Hits         1663     1663           
  Misses        126      126           
  Partials       81       81
```

**Comment by uweschaefer** [comment]
_From @uweschaefer on 2019-05-22 14:23:17 +0000 UTC_

closes #432
